src/components/auth/signup/DisplayMessage.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+const DisplayMessage = props => props.message && (
+  <div>
+    {
+      props.onSubmit
+      && (
+      <div className={typeof props.message === 'string' ? 'messageBox' : 'errorBox'}>
+        <div className="alert">
+          {typeof props.message === 'string' ? props.message
+            : props.message.map(error => <li key={error}>{error}</li>)}
+        </div>
+      </div>
+      )
+    }
+  </div>
+);
+
+export default DisplayMessage;
```

Comment:
Nice job on creating a component for this.

Reply:
Since there is a presentation component folder already. You should this to it instead of creating a new folder.